fedimint-core/src/util/mod.rs

```diff
@@ -320,41 +324,41 @@ pub fn handle_version_hash_command(version_hash: &str) {
 pub async fn retry<F, Fut, T>(
     op_name: impl Into<String>,
     op_fn: F,
-    interval: Duration,
-    max_attempts: u32,
+    statergy: impl backon::BackoffBuilder,
```

Comment:
Could we use this opportunity to switch the argument order? It's awkward to have the closure in the middle.

Reply:
updated